blog/2023-05-09-moving-to-matrix.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Chat is moving to Matrix
+authors:
+  - paddatrapper
+tags: [community]
+---
+
+Since late 2019, LibreTime has hosted our own Mattermost instance at
+https://chat.libretime.org. It has been used for project planning and some
+support requests. However, we want to make interacting with the project easy and
+not require yet another account, so we are moving to Matrix! The new community
+is available at [#libretime:matrix.org](https://matrix.to/#/#libretime:matrix.org).
+
+<!--truncate-->
+
+Mattermost is a very good open source platform for communication within an
+organisation, but it requires users to create an account specific for LibreTime.
+This is not ideal. It also requires us to maintain the instance and manage all
+the data therein.
+
+Matrix is an open network for secure, decentralized communication. There is a
+reference server provided by matrix.org that can be accessed from
+https://app.element.io, but people can also host their own homeserver and use
+that to interact. Conversations are split between rooms in one or more spaces. A
+space acts like an index for related spaces. We have a space
+(#libretime:matrix.org) with two rooms General (#libretime-general:matrix.org)
+and Support (#libretime-support:matrix.org). Any support queries should use the
+Support room.
+
+## Connecting to LibreTime Matrix rooms
+
+1. Create an account at https://app.element.io (if you don't have a Matrix
+   account already)
+2. Search for #libretime:matrix.org or open
+   https://matrix.to/#/#libretime:matrix.org.
+3. Join the Space and select the rooms you wish to join
+
+## Accessing the past Mattermost chat data
+
+TODO: Develop a method of accessing the chat log for the various rooms is
+accessible by the users in Mattermost without making it public.
+Password-protected?
```

Comment:
I think that's too much work, for not much added value.

What about sharing the data on user request, through an email request  (maybe don't print the email in the blog post, I hate spam :smile:) ?

Reply:
That would work I think